### PR TITLE
Week-3-PR-2: Add graph using a CSV file and the Google Charts Library

### PR DESCRIPTION
**Pull request by gracieorozco**
I added a graph using a CSV file with the Google Charts Library.

### CODE REVIEW

File: portfolio/src/main/webapp/projects.html
Comment (KidEinstein) [inline portfolio/src/main/webapp/projects.html]: I didn't find a matching opening tag for </p>. Shall we remove this?

File: portfolio/src/main/webapp/projects.html
Comment (KidEinstein) [inline portfolio/src/main/webapp/projects.html]: I don't see a matching opening tag for <a>. Did you want to make the URL a hyperlink?

File: portfolio/src/main/webapp/WEB-INF/ElectricityByCounty.csv
Comment (KidEinstein) [inline portfolio/src/main/webapp/WEB-INF/ElectricityByCounty.csv:44]: I live here. This was really good idea for a dataset for creating a chart. Good job!

File: portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java
Comment (KidEinstein) [inline portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java:32]: Let's add a javadoc above this class to briefly specify the use of this class.

**Review by KidEinstein — APPROVED** [review]
Thanks, Gracie! This PR looks good to me. Left a few minor comments and approving the PR. Please address the comments before merging. 

Also, make sure to merge this into master *after* you have merged week-3-pr-1 into master.

File: portfolio/src/main/webapp/projects.html
Comment (gracieorozco) [reply portfolio/src/main/webapp/projects.html]: Removed extra tag.

File: portfolio/src/main/webapp/projects.html
Comment (gracieorozco) [reply portfolio/src/main/webapp/projects.html]: Added extra opening tag.

File: portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java
Comment (gracieorozco) [reply portfolio/src/main/java/com/google/sps/servlets/GraphsServlet.java:32]: Added a javadoc.

**Review by KidEinstein — APPROVED** [review]
Thanks for updating!

**Pull request merged**
